pkg/console/subresource/configmap/brand_ocp.go

```diff
@@ -0,0 +1,7 @@
+// +build ocp
+
+package configmap
+
+const (
+	BRAND = "ocp"
```

Comment:
for ref, this `BRAND` will be the fallback for `getBrand()` from:
https://github.com/openshift/console-operator/pull/134/files#diff-bc08d2d514ac5156db922f45487af976R46

Reply:
Are you sure this is resolved?  I would have expected to see an update to `configmap.go` in this PR.